packages/react/vite.config.ts

```diff
@@ -9,34 +13,35 @@ import execute from "rollup-plugin-execute";
 // https://vitejs.dev/config/
 export default defineConfig(({ mode }) => {
   const env = loadEnv(mode, process.cwd(), "");
-  const CJS = env.BUILD_TARGET?.toLocaleLowerCase()?.match("cjs");
-  const formats: LibraryFormats[] = CJS ? ["cjs"] : ["es"];
+  const target = env?.BUILD_TARGET?.toLowerCase() as LibraryFormats;
 
   return {
     plugins: [
       dts({
         outDir: "dist/types",
       }),
       react(),
-      noBundlePlugin({ root: "./src" }),
+      // splitVendorChunkPlugin(),
+      noBundlePlugin({ root: resolve(__dirname, "src") }),
     ],
     build: {
-      outDir: CJS ? "dist/cjs" : "dist/esm",
+      // cssMinify: "lightningcss",
```

Comment:
Do we want to change css minifier? I didn't look too closely at the different options.

Reply:
Ah I was testing something out, isn't a big deal here, will just remove the commented code.